components/UI/Select/Select.tsx

```diff
@@ -0,0 +1,28 @@
+import { cn } from '@/utils';
+
+import { useSelect } from './hooks/useSelect';
+import { useSelectOptions } from './hooks/useSelectOptions';
```

Comment:
It would be better to create an `index.ts` file inside the `hooks` directory to simplify the import statements.